server_admin/topics/authentication/webauthn.adoc

```diff
@@ -167,27 +166,61 @@ A new user carries out the following operations :
+
+Administrator may typically require that Security Keys registered by users for the WebAuthn passwordless authentication must meet different
+(usually stronger) requirements. For example, those security keys may require users to authenticate to that security key using a PIN, or the
+security key should be attested with stronger certificate authority.
+
+Because of this situation, {project_name} allows administrator to configure separate `WebAuthn Passwordless Policy`. There is a separate required action
+of type `Webauthn Register Passwordless` and separate authenticator of type `WebAuthn Passwordless Authenticator`.
+
+===== Setup
+
+The setup procedure of WebAuthn passwordless support is the following :
+
+* Register new required action for WebAuthn passwordless support. Use the same steps as described <<_webauthn-register, above>>
+with the only difference, that you need to register the action called `Webauthn Register Passwordless`.
+
+* Configure the policy. You can use same steps and configuration options as described <<_webauthn-policy, above>>, however you
+need to configure them in the admin console in the tab `WebAuthn Passwordless Policy`. You can configure this policy as you want, however
+typically the requirements for the security key will be stronger than for the two-factor policy. For example the `User Verification Requirement` can
+be set to `Required` when you configure the passwordless policy.
+
+* Finally configure the authentication flow. Let's assume that we will use same flow called `WebAuthn Browser` as described
+<<_webauthn-authenticator-setup, above>>, but
+we will configure it as follows:
+
+** The `WebAuthn Browser Forms` subflow will contain `Username Form` as the first authenticator. This setting means that the user
+will provide just his or her username as the first step.
+
+** There will be a required subflow, which can be named for example `Passwordless Or Two-factor` . This setting indicates that user can
+authenticate either with Passwordless WebAuthn credential or with Two-factor authentication.
 
-A user carries out the following operations on the <<_account-service, `User Account Service`>>  :
+** Flow will contain `WebAuthn Passwordless Authenticator` as first alternative.
 
-- View the `Account` page.
+** The second alternative will be a subflow named for example `Password And Two-factor Webauthn`. This subflow will contain `Password Form` and
+`WebAuthn Authenticator`.
 
-===== Edit Registered WebAuthn Authenticator
+The full configuration of the flow will look like this:
 
-A user can edit the following information :
+image:images/webauthn-passwordless-flow.png[]
 
-- Label (WebAuthn authenticator's label the user entered on registering it)
+You can now add `WebAuthn Register Passwordless` as the required action to some user to test this. During the first authentication, the user will
```

Comment:
Maybe replace _as the required action to some user_ with _as the required action to some user, already known to {project_name},_ to test this? (This assumes Keycloak allows to set the `WebAuthn Register Passwordless` RA to some of the users, existing in Keycloak / already known to Keycloak. This differs e.g. from the `WebAuthn Register` RA action, which can be assigned solely to newly registered users AFAICT). 

But if the above assumption is wrong (IOW the new `WebAuthn Register Passwordless` RA can be only configured for newly registered Keycloak users too), I would complete remove the `to some user to test this.` part (IOW the sentence to be _You can now add a `WebAuthn Register Passwordless` as the required action for newly registered users.`

Reply:
Thanks, it is not specific to registered users. So I will use:
You can now add `WebAuthn Register Passwordless` as the required action to some user, already known to {project_name}, to test this.